sdk/appconfiguration/azure-appconfiguration/tests/test_azure_configuration_client_aad_async.py

```diff
@@ -44,9 +46,15 @@ def _delete_setting(self, client, item):
             key=item.key, label=item.label
         )
 
+    def create_aad_client(self, base_url):
+        cred = self.get_credential(AzureAppConfigurationClient, is_async=True)
+        client = AzureAppConfigurationClient(base_url, cred)
+        return AzureAppConfigurationClientProxy(client)
```

Comment:
just curious why this isn't the actual async client

Reply:
Xiang wrote a very clever class to take care of all the async features without having to change the base code. https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/appconfiguration/azure-appconfiguration/tests/async_proxy.py